Core/Core/View/Base/AppReview/AppReviewView.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  AppReviewView.swift
+//  Core
+//
+//  Created by  Stepanok Ivan on 26.10.2023.
+//
+
+import SwiftUI
+
+public struct AppReviewView: View {
+        
+    @ObservedObject private var viewModel: AppReviewViewModel
+
+    @Environment (\.isHorizontal) private var isHorizontal
+    @Environment (\.presentationMode) private var presentationMode
+    
+    public init(viewModel: AppReviewViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    public var body: some View {
+        ZStack {
+            Color.black.opacity(0.5)
+                .ignoresSafeArea()
+                .onTapGesture {
+                    presentationMode.wrappedValue.dismiss()
+                }
+            if viewModel.showSelectMailClientView {
+                SelectMailClientView(clients: viewModel.clients, onMailTapped: { client in
+                    viewModel.openMailClient(client)
+                })
+            } else {
+                VStack(spacing: 20) {
+                    if viewModel.state == .thanksForFeedback || viewModel.state == .thanksForVote {
```

Comment:
how about using `switch` here instead of if else?

Reply:
Good idea, done!